### ISSUE TITLE
Can't use cordova create --template

### ISSUE THREAD

**Issue by jbrown** [isleofcode/ember-cordova#100]
It would be great if you could pass a template through to the `cordova create` command.

```cli
$ ember install ember-cordova --template ../ember-cordova-template/
The option '--template' is not registered with the install command. Run `ember install --help` for a list of supported options.
 Could not install /Users/jb/code/ember-cordova-template
EISDIR: illegal operation on a directory, read
Error: EISDIR: illegal operation on a directory, read
    at Error (native)
```

**Comment by alexblom** [comment]
Agreed, thanks for bringing this up. 

I'll add support to this in the coming week or two, there are a few items in mid air right now.

**Comment by alexblom** [comment]
Fixed in 0.2.5
